tests/helix/send-to-helix-inner.proj

```diff
@@ -122,22 +122,6 @@
     <HelixPreCommand Condition="'$(OS)' != 'Windows_NT'" Include="export PATH=$HELIX_CORRELATION_PAYLOAD/azure-func-cli:$PATH" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(HasDocker)' == 'true'">
-    <HelixPreCommand Include="docker info" />
-    <HelixPreCommand Include="docker ps" />
-    <HelixPreCommand Include="docker container ls --all" />
-    <HelixPreCommand Include="$(_ShutdownDockerContainersCommand)" />
-    <HelixPreCommand Include="docker volume ls" />
-    <HelixPreCommand Include="$(_DeleteDockerVolumesCommand)" />
-    <HelixPreCommand Include="docker network prune -f" />
-
-    <HelixPostCommand Include="docker container ls --all" />
```

Comment:
This is useful in situations where the container crashes, to see what is happening on the machine. It is also nice to guarantee somewhat stable machines.

Do we have to delete this?

cc @radical

Reply:
While that is true, it's nicer if we can actually find bugs instead of having the CI cover up dcp problems 😄

Reply:
Can we at least add back the diagnostics lines? If you don't want the prune/clean calls for now, that's fine.

Reply:
Yep that's reasonable

Reply:
@karolz-ms can you add the diagnostic ones back?

Reply:
Working on it

Reply:
https://github.com/dotnet/aspire/pull/7141